app/bundles/LeadBundle/Controller/Api/LeadApiController.php

```diff
@@ -119,6 +119,29 @@ public function getOwnersAction()
     }
 
     /**
+     * Obtains a specific entity url history of specific contact
+     *
+     * @param int $id contact id
+     *            
+     * @return Response
+     */
+    public function getEntityUrlHistoryAction($id)
+    {
+        $start= $this->request->query->get('start', 0);
+        $limit= $this->request->query->get('limit', 0);
+        
+        $entity = $this->model->getEntityUrlHistory($id,$start,$limit);
```

Comment:
Is there a case for using only page hits with this endpoint? Thoughts on leveraging the filterable timeline instead? 

https://github.com/mautic/mautic/blob/6b32de67cab12b23eff1fbaff2ae770c4eafe625/app/bundles/LeadBundle/Controller/LeadDetailsTrait.php#L95